gamemode/modules/base/cl_entityvars.lua

```diff
@@ -7,10 +7,11 @@ local pmeta = FindMetaTable("Player")
 -- This function is made local to optimise getDarkRPVar, which is called often
 -- enough to warrant optimizing. See https://github.com/FPtje/DarkRP/pull/3212
 local get_user_id = pmeta.UserID
-function pmeta:getDarkRPVar(var)
+function pmeta:getDarkRPVar(var, fallback)
+    fallback = fallback or nil
```

Comment:
Change this to an explicit nil check, otherwise fallback will be forced to nil if it's false.

Reply:
Thx